### DISCUSSION TITLE
Is there a way to export raw Custom events?

### DISCUSSION THREAD

**Discussion by jchatard** [plausible/analytics#1660] [Anything Else]
Hi,

Given some custom events that I send using the JS client:

```js
plausible('show', {props: {template: 'page', zone: 'header', context: 'invest'}})
plausible('show', {props: {template: 'page', zone: 'footer', context: 'dwell'}})
plausible('submit', {props: {template: 'page', zone: 'footer', context: 'dwell'}})
```

Is there a way to export all events along their custom props (as JSON I guess, or CSV might be possible too) for a given period of time?

If my understanding is correct that is not possible using the breakdown API.

Thank you :-)
Jérémy

**Comment by metmarkosaric** [comment]
hi! have you looked at [our CSV export](https://plausible.io/docs/export-stats) feature? if you filter the dashboard by a custom event and then use the export, there should be a prop_breakdown.csv file within the export

**Reply by jchatard** [reply]
Hmm... That's not exatcly what I was expecting format-wise. But it _may_ be helpful to what I'm trying to achieve.

Many thanks, 
I'll have a look!

Cheers,

**Reply by fr3fou** [reply]
Is there a way to programatically export custom events without using the dashboard?
